utils/src/main/java/com/cloud/utils/net/NetUtils.java

```diff
@@ -420,30 +422,49 @@ public static String[] getNicParams(final String nicName) {
         }
     }
 
-    public static String[] getNetworkParams(final NetworkInterface nic) {
+    public static String[] getNetworkParams(NetworkInterface nic) {
+        s_logger.debug(String.format("Retrieving network params of NIC [%s].", nic));
+
+        s_logger.trace(String.format("Retrieving all NIC [%s] addresses.", nic));
```

Comment:
These logs are redundant

Reply:
Actually the first log refers to whole process and the second to the step being executed.